sweep/sweeper.go

```diff
@@ -188,6 +201,9 @@ type pendingInput struct {
 	// lastFeeRate is the most recent fee rate used for this input within a
 	// transaction broadcast to the network.
 	lastFeeRate chainfee.SatPerKWeight
+
+	// rbf records the RBF constraints.
+	rbf *RBFInfo
```

Comment:
I think this is another instance where we want to use Option instead of a pointer.

Reply:
make sense, switched.

Reply:
Actually a second thought is, I think it makes more sense to use Option when the empty value means something so we can distinguish between an empty value and a not-set value?

Reply:
I view those two cases as the same. The point is that if we want to progressively adopt the approach that nil pointers are error/invalid states then any version of a missing value that is allowed and expected should be an option.